lib/Target/CBackend/CBackend.cpp

```diff
@@ -1689,7 +1689,7 @@ void CWriter::writeInstComputationInline(Instruction &I) {
   Type *Ty = I.getType();
   if (Ty->isIntegerTy()) {
     IntegerType *ITy = static_cast<IntegerType *>(Ty);
-    if (!ITy->isPowerOf2ByteWidth())
+    //if (!ITy->isPowerOf2ByteWidth())
```

Comment:
This looks like a hack.

Reply:
it was because at the time I didnt have a replacement for that function